### PR TITLE
Add sprint nights runbook.

### PR DESCRIPTION
**Pull request by geeksambhu**
This PR includes a runbook for sprint nights.

### CODE REVIEW

**Comment by sahmed007** [comment]
LGTM

**Review by dijital20 — APPROVED** [review]
Looks like a good start. Let's make sure we take another look at it and adjust after we've gotten a few under our belt. :)

File: docs/meetup/runbooks/sprint-nights.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Sprint Nights Runbook
+
+## Objective
+To organize a weekly Sprint nights for PyTexas pythonistas to supplement the existing monthly meetup on Discord.
+
+## Steps
+
+### Identify Key Details
+- **Frequency:** Weekly
```
Comment (MasonEgger) [inline docs/meetup/runbooks/sprint-nights.md]:
```suggestion
- **Frequency:** Monthly
```

File: docs/meetup/runbooks/sprint-nights.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- **Day & Time:** [e.g., Every Thursday at 9 PM]
```
Comment (MasonEgger) [inline docs/meetup/runbooks/sprint-nights.md]:
```suggestion
- **Day & Time:** [e.g., Thursday at 7 PM]
```

**Review by MasonEgger — CHANGES_REQUESTED** [review]

**Review by MasonEgger — APPROVED** [review]

**Pull request merged**